app/src/ui/no-repositories/create-tutorial-repository-dialog.tsx

```diff
@@ -184,7 +184,7 @@ export class CreateTutorialRepositoryDialog extends React.Component<
 
       if (await pathExists(path)) {
         throw new Error(
-          `The path ${path} already exists. Please move it ` +
+          `The path '${path}' already exists. Please move it ` +
```

Comment:
I don't think this full path is very helpful, and will get sent to central as a non-fatal error. Does adding the `''` make it easier to scrub? i think we could also:
```suggestion
          `The path '${name}' already exists. Please move it ` +
```

Reply:
> I don't think this full path is very helpful

While I agree that it's not super helpful for us this exception message is presented as-is to the user and for them I'd argue it's quite necessary to see the full path. In fact, `name`, here is a constant (`'desktop-tutorial'`). Until we've built in support for overwriting, or forcing the repo creation elsewhere, we need the user to be able to go into explorer/finder and manually remove the tutorial folder.

> Does adding the '' make it easier to scrub?

I wan initially thinking we should add a generic scrubber for anything matchin `path '[^']'` but I've added a specific scrub for this pattern instead. For consistency though I think we should keep it, we usually put quotes around paths in exception messages.

Reply:
@niik Ah right. The path is useful for the user, but not the exception reporting.

> but I've added a specific scrub for this pattern instead

So that means this particular error message will be scrubbed in `central`? we could also add logic in the `catch` statement to remove the path (near line 226).

Reply:
> So that means this particular error message will be scrubbed in `central`?

Yep, that part is being deployed as we speak!

> we could also add logic in the `catch` statement to remove the path (near line 226).

Yeah, we could but I need to scrub it for versions existing already so the central side was needed regardless. Additionally I think we should discuss not sending these exceptions to the non-fatal bucket any more. We've captured enough data to understand what the problem is and I think we should treat the non-fatal bucket as just that a temporary resource to investigate a problem (rather than a catch-all bucket).

Reply:
sounds good @niik! thank you!